doc/mgmt/Management Framework.md

```diff
@@ -651,63 +845,111 @@ REST server will accept concurrent requests. Translib provides appropriate locki
 
 ###### 3.2.2.4.6 API Versioning
 
-REST server will allow clients to specify API version through a custom HTTP header "Accept-Version". However API versioning feature will be supported only in a future release. The server will ignore the version information in current release.
+REST server will allow clients to specify API version through a custom HTTP header **Accept-Version**.
 
-    Accept-Version: 2019-06-20
     Accept-Version: 1.0.3
 
-REST server will extract version text from the request header and pass it to the Translib API as metadata. App modules can inspect the version information and act accordingly.
+Version text should be in **MAJOR.MINOR.PATCH** format. REST server will extract version text from
+the request header and pass it to the Translib API as **ClientVersion** argument.
+Section [3.2.2.6.4.2](#322642-version-checks) explains Translib version checking logic.
+
+Version checks are bypassed if Accept-Version header is not present in the request.
 
-For YANG defined RESTCONF APIs, the version is the latest YANG revision date. For manual OpenAPI definitions developer can define version text in any appropriate format.
+For YANG defined RESTCONF APIs, the server's version can be discovered through the standard YANG
+module library API "GET /restconf/data/ietf-yang-library:modules-state/module-set-id".
+Client should not pass Accept-Version header for this API! This is safe since module-set-id
+is a standard API defined by [RFC7895](https://tools.ietf.org/html/rfc7895) for this very purpose.
+
+API Versioning is not supported for manual OpenAPI definitions in this release.
+Accept-Version header value will be ignored even if specified.
 
 ###### 3.2.2.4.7 RESTCONF Entity-tag
 
-REST server will support RESTCONF entity-tag and last-modified timestamps in next release. server will not process or send corresponding request, response headers in first release.
-Note that entity-tag and last-modified timestamps will be supported only for top level datastore node (/restconf/data). Per resource entity tags and timestamps will not be supported. Global entity tag and timestamp are used for configuration resources.
+REST server does not maintain entity-tag and last-modified timestamp for configuration data nodes.
+GET and HEAD responses does not include "ETag" and "Last-Modified" headers.
```

Comment:
what is the significance of ETag  & Last-Modified exclusion from the rest URLs?

Reply:
REST client cannot use RFC7232 style conditional requests now. It would have been useful for clients that cache the server data. Conditional requests allow clients to get data only if it is modified or set only if not modified etc..